build.sbt

```diff
@@ -433,7 +433,7 @@ lazy val benchmarks = project.module
         "org.scala-lang"             % "scala-reflect"  % scalaVersion.value,
         "org.typelevel"             %% "cats-effect"    % "2.2.0",
         "org.scalacheck"            %% "scalacheck"     % "1.15.1",
-        "qa.hedgehog"               %% "hedgehog-core"  % "0.6.0",
+        "qa.hedgehog"               %% "hedgehog-core"  % "0.5.1",
```

Comment:
`0.6.0` version of `hedgehog-core` is nowhere to be found... It seems like it's been yanked from Maven Central